docs/guide/installation.md

```diff
@@ -64,7 +64,7 @@ If you are using `enableStrictParsing` URL manager option, add the following to
 ### Extra configuration for logging and profiling
 
 Logging and profiling are simple but powerful tools that may help you to understand the execution flow of both the
-framework and the application. These tools are useful for development and production environments alike.
+framework and the application. These tools are useful for development and production environments NOT alike.
```

Comment:
It says that logging and profiling could be useful for both development and production which is true. It doesn't say to log everything in production.